### ISSUE TITLE
Subscription Support

### ISSUE THREAD

**Issue by HuakunShen** [graphql-rust/graphql-client#448]
Is subscription fully supported? 

In the README.md I saw `Subscriptions support (serialization-deserialization only at the moment).`

Not sure what this means. Does it mean subscription is not yet supported?

**Comment by tomhoule** [comment]
Let's document [graphql-ws-client](https://crates.io/crates/graphql-ws-client) in the README, and leave this out of scope for graphql-client, if that makes sense. I'm trying to refocus the crate on codegen only.

**Comment by Sytten** [comment]
Yes it works well since I integrated it with graphql-client. Been using in prod for over 2 years now.
